LCLSGeneral/LCLSGeneral/GVLs/Constants.TcGVL

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <GVL Name="Constants" Id="{57f2f1ff-0c58-4e70-a147-fb0d6c2bb252}">
+    <Declaration><![CDATA[
+VAR_GLOBAL CONSTANT
```

Comment:
Any thoughts about "qualified only" access here and the overall name "Constants"? (`{attribute 'qualified_only'}` before `VAR_GLOBAL`)

I personally do not like global constants without a namespace prefix appearing in our projects. I believe (correct me if I'm wrong) the following three could be valid without 'qualified only':

```
iValue := LCLS_General.Constants.MAX_STATES;
iValue := Constants.MAX_STATES;
iValue := MAX_STATES;
```

Reply:
You are correct that all of those would be valid as-is.

I omitted "qualified_only" on this because the other general lib global variables also omitted it. It's nice having a single canonical way to refer to something and it's not easy to go back to qualified_only once you've skipped it. I wouldn't mind changing it.

Are you thinking qualified_only + a more specific GVL name, like "GeneralConstants.MAX_STATES"?

For the record I don't have a super strong opinion here.

Reply:
> Are you thinking qualified_only + a more specific GVL name, like "GeneralConstants.MAX_STATES"?

Yes, that's the direction I was aiming for and wanted to hear your opinion.